sormas-backend/src/test/java/de/symeda/sormas/backend/contact/ContactFacadeEjbTest.java

```diff
@@ -1750,4 +1866,27 @@ public void testGetContactUsersWithoutUsersLimitedToOthersDiseses() {
 		Assert.assertTrue(userReferenceDtos.contains(limitedCovidNationalUser));
 		Assert.assertFalse(userReferenceDtos.contains(limitedDengueNationalUser));
 	}
+
+	private CaseDataDto getCaze(UserDto user, PersonDto cazePerson, RDCFEntities rdcf) {
```

Comment:
this is a bit misleading, would be better to rename these methods to createCaze...